src/planscape/plan/models.py

```diff
@@ -88,6 +88,8 @@ class Scenario(models.Model):
 
     project = models.ForeignKey(
         Project, on_delete=models.CASCADE, null=True)  # type: ignore
+    
+    notes: models.TextField = models.TextField(null=True)
```

Comment:
nit: can the assignment format be consistent in this file? Some of these fields have type hints while others don't.

Reply:
I think this is intentional, they are only assigned for FK fields.